quest/mp2.py

```diff
@@ -0,0 +1,81 @@
+"""
+MP2 This is a sample mp2 file for testing.
+"""
+import time
+import numpy as np
+import psi4
+
+
+def mp2(mol, basis="aug-cc-pvdz"):
+
+    print('\nStarting MP2...')
+    # Memory for Psi4 in GB
+    psi4.set_memory('2 GB')
+    psi4.core.set_output_file('output.dat', False)
+
+    # Memory for numpy in GB
+    numpy_memory = 2
+
+    psi4.set_options({'basis': basis,
+                      'scf_type': 'pk',
+                      'mp2_type': 'conv',
+                      'e_convergence': 1e-8,
+                      'd_convergence': 1e-8})
+
```

Comment:
Please do not set psi4 options, you will need to use the options dictionary from the Wavefunction.